lib/node/runner/api/transaction_post.go

```diff
@@ -0,0 +1,59 @@
+package api
+
+import (
+	"boscoin.io/sebak/lib/error"
+	"boscoin.io/sebak/lib/network/httputils"
+	"boscoin.io/sebak/lib/node/runner/api/resource"
+	"boscoin.io/sebak/lib/transaction"
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+)
+
+type TeeReadCloser struct {
+	io.ReadCloser
+	teeReader io.Reader
+}
+
+func (tee TeeReadCloser) Read(p []byte) (n int, err error) {
+	return tee.teeReader.Read(p)
+}
+
+func NewTeeReadCloser(origin io.ReadCloser, w io.Writer) io.ReadCloser {
+	return &TeeReadCloser{
+		ReadCloser: origin,
+		teeReader:  io.TeeReader(origin, w),
+	}
+}
+
+func (api NetworkHandlerAPI) PostTransactionsHandler(w http.ResponseWriter, r *http.Request, handler http.HandlerFunc) {
```

Comment:
It is the most important change in this PR.
It intercepts the input/output of the handler.
And execute the handler.
After that, wraps the output with additional information

Reply:
@kfangw One question; Did you intend to re-use the `runner.MessageHandler()` for `PostTransactionHandler()`? If so, how about implement copy-of `runner.MessageHandler()`. Code can be duplicated, but this intercept thing will be removed, code will be simpler. :)